src/interfaces/javaapi/javaapi_secured.go

```diff
@@ -116,7 +116,7 @@ const (
 	platform      = "android"
 	executionType = "android"
 
-	edgeDir = "/data/user/0/com.samsung.orchestration.service/files"
+	edgeDir = "/data/user/0/com.samsung.homeedge/files"
```

Comment:
How about moving this variable to input parameter on javaapi?

Reply:
Yes, as we discussed about this issue, I think it's better to separate this value from edge-orchestration module.
It doesn't look good to place this constant value in orchestration module.
I'll check and prepare with better method.

Reply:
Thanks :+1: